controllers/tests/controller_server/host_definer/csi_node_watcher_test.py

```diff
@@ -0,0 +1,322 @@
+from unittest.mock import Mock
+from kubernetes.client.rest import ApiException
+
+import controllers.tests.controller_server.host_definer.utils.utils as utils
+import controllers.tests.controller_server.host_definer.settings as settings
+from controllers.servers.host_definer.types import DefineHostResponse
+import controllers.servers.host_definer.messages as messages
+from controllers.tests.controller_server.host_definer.common import BaseSetUp
+
+
+class TestAddInitialCsiNodes(BaseSetUp):
+    def test_is_host_can_be_defined_not_called_when_there_is_csi_node_but_not_with_ibm_block_provider(self):
+        self.csi_node_watcher._is_host_can_be_defined = Mock()
+        self.csi_node_watcher.csi_nodes_api.get.return_value = self.fake_k8s_csi_nodes_without_ibm_block
+        self.csi_node_watcher.add_initial_csi_nodes()
+        self.csi_node_watcher._is_host_can_be_defined.assert_not_called()
```

Comment:
let's check the actual method that adds the host and/or the NODES set?

Reply:
same for all privet method calls

Reply:
done